16_FirstContribution/StonePaperScissors

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+import java.util.Random;
+
+class spsgame {
+     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        Scanner ex = new Scanner(System.in);
```

Comment:
Defining two scanner objects is unnecessary. Only one scanner object can be used for all inputs

Reply:
I removed one but it didn't change while submitting